### PR TITLE
[DOCS] Added RNs for 6.3.2

### PR DESCRIPTION
**Pull request by debadair**
Ran the script to generate the release notes for 6.3.2.

### CODE REVIEW

File: docs/reference/release-notes/6.3.asciidoc [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+* Disable C2 from using AVX-512 on JDK 10 {pull}32138[#32138] (issue: {issue}31425[#31425])
+
+Machine Learning::
+* Fix calendar and filter updates from non-master nodes {pull}31804[#31804] (issue: {issue}31803[#31803])
+* Don't treat stale FAILED jobs as OPENING in job allocation {pull}31800[#31800] (issue: {issue}31794[#31794])
+* Rate limit established model memory updates {pull}31768[#31768]
+* Wait for aliases in multi-node tests {pull}32086[#32086] (issue: {issue}32034[#32034])
+
+Mapping::
+* Fix put mappings java API documentation {pull}31955[#31955] (issue: {issue}31906[#31906])
```
Comment (pcsanwald) [inline docs/reference/release-notes/6.3.asciidoc]: I think we should remove this: it's a docs change, but the PR was not labelled as such. I've pinged @cbuescher for a second opinion, but I think we should remove this and label #31955 with `>docs`

**Review by pcsanwald — CHANGES_REQUESTED** [review]
There's one item I think we should remove. the rest LGTM

**Comment by lcawl** [comment]
I double-checked with the PR owner and https://github.com/elastic/elasticsearch/pull/32086 should be omitted from the list too.  It's just a test-related change.

**Comment by debadair** [comment]
Removed the doc & test only items.

**Review by pcsanwald — APPROVED** [review]
LGTM

**Pull request merged**